mbta-performance/chalicelib/historic/process.py

```diff
@@ -0,0 +1,53 @@
+import pandas as pd
+import pathlib
+
+
+def process_events(input_csv: str, outdir: str, nozip: bool = False):
+    columns = [
```

Comment:
we're redefining and re-listing column names that are pretty similar to each other in many different places across the repo. perhaps the column names should be moved to the top of this file (as something like `HISTORIC_COLUMNS`) or to `contants.py`?

Reply:
Good point, done